src/main/java/Sirius.java

```diff
@@ -0,0 +1,148 @@
+import java.util.Scanner;
+import java.lang.Integer;
+import java.util.ArrayList;
+public class Sirius {
+    // some commands
+    public static final String BYE = "bye";
+    public static final String LIST = "list";
+    public static final String MARK = "mark";
+    public static final String UNMARK = "unmark";
```

Comment:
is this really needed? you only used this once in the Sirius.java file (in the switch case portion). while it does look neat below, I don't think it is really necessary